eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Epoch.hs

```diff
@@ -193,11 +173,13 @@ epochTransition = do
       TRC (epochState', UpecState pp (utxosPpups utxoSt'), ())
   let utxoSt'' = utxoSt' {utxosPpups = ppupSt'}
 
-  let Coin oblgCurr = obligation pp (rewards dstate') (psStakePoolParams pstate'')
-      Coin oblgNew = obligation pp' (rewards dstate') (psStakePoolParams pstate'')
+  let -- Coin oblgCurr = obligationDPState (DPState dstate' pstate'')
```

Comment:
I guess this particular comment is left by accident here ?

Reply:
yes, I will remove it.